src/api/app/assets/javascripts/webui/application.js

```diff
@@ -36,6 +36,7 @@
 //= require webui/application/attribute
 //= require webui/application/main
 //= require webui/application/repository_tab
+//= require webui/application/moment
```

Comment:
yep, just a few changes: server and client might be in different timezone, so we changed time to UTC. Also if server time is a little ahead of client, it might show the time difference "in future" even though you have just made changes there. So we changed the "in future" to "now".